packages/color-field/src/ColorField.ts

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import {
+    CSSResultArray,
+    html,
+    TemplateResult,
+} from '@spectrum-web-components/base';
+
+import { property } from '@spectrum-web-components/base/src/decorators.js';
+import { TinyColor } from '@ctrl/tinycolor';
+import '@spectrum-web-components/textfield/sp-textfield.js';
+import { TextfieldBase } from '@spectrum-web-components/textfield';
+
+//import styles from './color-field.css.js';
+
+/**
+ * @element sp-color-field
+ */
+export class ColorField extends TextfieldBase {
+    public static override get styles(): CSSResultArray {
+        return [TextfieldBase.styles];
+    }
+
+    @property({ type: Boolean, reflect: true })
+    public viewColor = false;
```

Comment:
Do we really want `viewColor` to always be available to consumers?

There is no custom CSS that `[viewColor]` can manage, do we really want to reflect this value?

Reply:
I think we should make viewColor available to consumers, it wont be available by defauld but if a consumer wants to use this viewColor feature, it should he available. We can ask CSS team for help with this!